From 2226b506aa0ae62a689419e19b40be2135ad2529 Mon Sep 17 00:00:00 2001
From: Nikola Pajkovsky <npajkovs@redhat.com>
Date: Tue, 28 Feb 2012 11:43:02 +0100
Subject: [PATCH 030/105] koops-unit-test: bundle test koops into tarball

Signed-off-by: Nikola Pajkovsky <npajkovs@redhat.com>
---
 Makefile.am | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/Makefile.am b/Makefile.am
index 6599b64..07e4cf5 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -11,6 +11,10 @@ TESTSUITE_FILES += examples/oops-kernel-3.x.x
 TESTSUITE_FILES += examples/oops1.right
 TESTSUITE_FILES += examples/koops-tainted-bg1
 TESTSUITE_FILES += examples/oops-same-as-oops4.right
+TESTSUITE_FILES += examples/oops4.right
+TESTSUITE_FILES += examples/oops-same-as-oops4.right
+TESTSUITE_FILES += examples/hash-gen-oops6.right
+TESTSUITE_FILES += examples/hash-gen-same-as-oops6.right
 
 EXTRA_DIST = doc/coding-style abrt.spec.in abrt.pc.in \
 	abrt-version asciidoc.conf init-scripts/* $(TESTSUITE_FILES)
-- 
1.7.11.2

