From 3a6c5cd611cc1272fe04eb058a017c8c02383e43 Mon Sep 17 00:00:00 2001
From: Nikola Pajkovsky <npajkovs@redhat.com>
Date: Thu, 3 May 2012 15:48:58 +0200
Subject: [PATCH 080/105] applet: reverse list to not be surprise during
 debugging

Signed-off-by: Nikola Pajkovsky <npajkovs@redhat.com>
---
 src/applet/applet.c | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/src/applet/applet.c b/src/applet/applet.c
index ddf8cb0..b6178ec 100644
--- a/src/applet/applet.c
+++ b/src/applet/applet.c
@@ -85,7 +85,7 @@ static GList *add_dirs_to_dirlist(GList *dirlist, const char *dirname)
     }
     closedir(dir);
 
-    return dirlist;
+    return g_list_reverse(dirlist);
 }
 
 /* Return 1 if a new directory appeared, compared to list saved
@@ -122,6 +122,7 @@ static int new_dir_exists(void)
         while ((line = xmalloc_fgetline(fp)) != NULL)
             old_dirlist = g_list_prepend(old_dirlist, line);
 
+        old_dirlist = g_list_reverse(old_dirlist);
         /* We will sort and compare current dir list with last known one.
          * Possible combinations:
          * DIR1 DIR1 - Both lists have the same element, advance both ptrs.
-- 
1.7.11.2

