From 5637b6f77887509526beb706844b8c43d4ce4969 Mon Sep 17 00:00:00 2001
From: Jiri Moskovcak <jmoskovc@redhat.com>
Date: Tue, 8 Jan 2013 15:39:50 +0100
Subject: [PATCH 124/124] libabrt: link with libreport rhbz#892658

Signed-off-by: Nikola Pajkovsky <npajkovs@redhat.com>
---
 src/lib/Makefile.am | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/src/lib/Makefile.am b/src/lib/Makefile.am
index bbeb687..6d6c0d0 100644
--- a/src/lib/Makefile.am
+++ b/src/lib/Makefile.am
@@ -23,4 +23,5 @@ libabrt_la_CPPFLAGS = \
 libabrt_la_LDFLAGS = \
     -version-info 0:1:0
 libabrt_la_LIBADD = \
-    $(GLIB_LIBS)
+    $(GLIB_LIBS) \
+    $(LIBREPORT_LIBS)
-- 
1.8.0.1

