From 29a9a7d009b501a8ef9e6ef023f336bd0a6732cc Mon Sep 17 00:00:00 2001
From: Denys Vlasenko <vda.linux@googlemail.com>
Date: Mon, 28 Jan 2013 09:56:40 +0100
Subject: [PATCH 132/132] abrtd: disable glib's buffering on inotify reads.
 Closes rhbz#873815

Signed-off-by: Denys Vlasenko <vda.linux@googlemail.com>
---
 src/daemon/abrtd.c | 7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/src/daemon/abrtd.c b/src/daemon/abrtd.c
index 6faaedc..f54cd61 100644
--- a/src/daemon/abrtd.c
+++ b/src/daemon/abrtd.c
@@ -948,6 +948,13 @@ int main(int argc, char** argv)
      */
     ndelay_on(inotify_fd);
     channel_inotify = my_io_channel_unix_new(inotify_fd);
+    /*
+     * glib's read buffering must be disabled, or else
+     * FIONREAD-reported "available data" sizes and sizes of reads
+     * can become inconsistent, and worse, buffering can split
+     * struct inotify's (very bad!).
+     */
+    g_io_channel_set_buffered(channel_inotify, false);
     channel_inotify_event_id = g_io_add_watch(channel_inotify,
                      G_IO_IN | G_IO_PRI | G_IO_HUP,
                      handle_inotify_cb,
-- 
1.8.1

